src/app/sb-admin/components/pages/sb-user/sbuser.component.html

```diff
@@ -51,7 +51,7 @@ <h5 class="m-0">{{ 'USER' | i18next }}</h5>
                             <p-tableCheckbox [value]="user"></p-tableCheckbox>
                         </td>
                         <td ><span class="p-column-title">Organization</span>
-                            {{user.rootOrgName}}
+                            {{user.channel}}
                         </td>
                         <td><span class="p-column-title">First Name</span>
```

Comment:
Language constant is missing. remove all the hardcoded Texts from the HTML file

Reply:
Resolved